standard/circuit_tier_highlight/commons/highlight_conditions.lua

```diff
@@ -0,0 +1,33 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:HighlightConditions
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+
+local HighlightConditions = {}
+
+--- Check arguments or queryData if the tournament should be highlighted
+---@param data table
+---@param options table
+---@return boolean
+function HighlightConditions.tournament(data, options)
+	data.extradata = data.extradata or {}
+	options = options or {}
+
+	if options.onlyHighlightOnValue then
+		return data.publishertier == options.onlyHighlightOnValue
+	end
+
+	return Logic.nilOr(
+		Logic.readBoolOrNil(data.publishertier),
+		String.isNotEmpty(data.extradata.publisherpremier)
+			or (String.isNotEmpty(data.extradata.valvepremier) and data.publishertier ~= 'Minor')
```

Comment:
IMO check which wikis actually need this and move to their custom to not carry over legacy behavior into this standardized module. It feels like the valorant or overwatch version should actually be the new commons one.

Reply:
looking at the infoboxes only dota has `extradata.valvepremier` set and they use `String.isNotEmpty(args.valvepremier) and '1' or '0'` so it is always set for them ...

`extradata.publisherpremier` on the other hand is set in prize pools by
- dota
- lol
- ml
- pokemon

and apparently they don't hold the same values as `publishertier`

so imo:
- add dota custom and kick line 29 from commons
- keep line 28

Reply:
Seems reasonable